### ISSUE TITLE
Reference node for single/small monitors and easy comparing

### ISSUE THREAD

**Issue by UnrealPinball** [njbrown/texturelab#35]
A node that just holds a reference image. You can scale the size and just move it around easily.
![sdgsdvs](https://user-images.githubusercontent.com/55934872/116770330-5dbdee00-aa86-11eb-96aa-66dbef08b1db.PNG)

**Comment by BladiCreator** [comment]
you can do it with pureref

**Comment by CreatorSiSo** [comment]
Yeah i dont think you need that inside tl (although having an reference image in the 2D View at some point might be cool)

**Comment by UnrealPinball** [comment]
Mostly for small screens or maybe in the future tablets, hard to use reference on the side if that's the case.

**Comment by leoschwarz** [comment]
There are already image nodes which you can use for this purpose too. But resizing nodes would be a good feature to have I think.
